tools/shared/src/main/scala/org/scalajs/core/tools/linker/analyzer/Analyzer.scala

```diff
@@ -165,6 +173,7 @@ private final class Analyzer(semantics: Semantics,
   private class ClassInfo(data: Infos.ClassInfo) extends Analysis.ClassInfo {
     private[this] var _linking = false
```

Comment:
I can't. That's what I tried initially, but I failed to discover cycles and it went into infinite loop. There is a time during which `_linking` is `true` but `_linkingFrom` needs to stay `null`.